explainability-core/src/test/java/org/kie/trustyai/metrics/utils/ArrayGenerators.java

```diff
@@ -0,0 +1,107 @@
+package org.kie.trustyai.metrics.utils;
```

Comment:
Move out of `test` dir

Reply:
I'm not sure these are needed outside of the tests